tests_e2e/lisa/tests/agent_bvt/check_agent_version.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
Skeleton for the current scenarios\agent-bvt\01_check_waagent_version.py test in DCR